cli/src/main/java/org/adamalang/cli/router/OutputTableStream.java

```diff
@@ -0,0 +1,8 @@
+package org.adamalang.cli.router;
+
+import com.fasterxml.jackson.databind.node.ObjectNode;
+
+public interface OutputTableStream {
```

Comment:
Is this used? If not, then delete it (ctrl-f here showed me nothing). Is this generated? If so, remove the generator